onadata/libs/utils/export_tools.py

```diff
@@ -158,6 +158,9 @@ def generate_export(export_type, xform, export_id=None, options=None):
         records = records.iterator()
 
     export_builder = ExportBuilder()
+    reviews = Instance.objects.filter(xform=xform, has_a_review=True).count()
```

Comment:
Should we have the option to export reviews something that is an export option rather than check the database for submissions with reviews?